applications/sudokumb/WebApp/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,155 @@
+@{
+// Copyright (c) 2018 Google LLC.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+}
+
+@{
+    ViewData["Title"] = "Home Page";
+}
+<style>
+    dd {padding-bottom: 1em}
+</style>
+
+<div class="row">
+    <div class="col-md-6">
+        <h1>Sudoku + Dumb = Sudokumb</h1>
+        <p>
+            <strong>Sudokumb</strong> is a
+            <a href="https://en.wikipedia.org/wiki/Sudoku">Sudoku</a> solver.
+            In order to demonstrate some cool features of
+            <a href="https://cloud.google.com/">Google Cloud Platform
+            <img src="http://cloud.google.com/_static/images/cloud/products/logos/svg/gcp.svg"
+                width=16 /></a>,
+            Sudokumb solves the puzzles in a very dumb way.
+            But some things it does are actually pretty smart.
+        </p>
+        <form asp-controller="Sudokumb" asp-action="Index" method=GET>
+            <button type="submit" class="btn btn-primary">Try it!</button>
+        </form>
+    </div>
+    <div class="col-md-4" style="padding-top: 1em">
+        <pre>1 2 3   6 4 5   7 8 9
+9 8 7   3 2 1   6 5 4
+6 4 5   9 8 7   3 1 2
+
+8 3 9   4 7 2   5 6 1
+2 7 6   1 5 9   8 4 3
+5 1 4   8 3 6   9 2 7
+
+7 9 1   2 6 8   4 3 5
+4 5 8   7 1 3   2 9 6
+3 6 2   5 9 4   1 7 8</pre>
+    </div>
+</div>
+
+
+<div class="row">
+    <div class="col-md-5">
+        <h2>Smart</h2>
+        <dl>
+            <dt>
+                <img src="http://cloud.google.com/_static/images/cloud/products/logos/svg/datastore.svg"
+                    width=64/>
+                Sudokumb stores user info in Datastore.
+            </dt>
+            <dd>
+                Sudokumb stores user information like email addresses and passwords in 
+                <a href="https://cloud.google.com/datastore/">Google Cloud Datastore></a>.
```

Comment:
Update to remove orphaned ">" symbol after "Datastore"

Reply:
Done.